test-requirements.txt

```diff
@@ -7,3 +7,4 @@ coverage>=3.6
 mock>=1.0.1
 docker-py>=1.1.0
 path.py>=5.2
+testrepository>=0.0.20
```

Comment:
Each time there is a modification here, we need to align the deps for packages. At least create a ticket so I know and can do it.